docs/sources/features/datasources/azuremonitor.md

```diff
@@ -331,21 +366,11 @@ If you're not currently logged in to the Azure Portal, then the link opens the l
 
 Grafana alerting is supported for Application Insights. This is not Azure Alerts support. Read more about how alerting in Grafana works in [Alerting rules]({{< relref "../../alerting/alerts-overview.md" >}}).
 
-### Writing analytics queries For the Application Insights service
-
-If you change the service type to "Application Insights", the menu icon to the right adds another option, "Toggle Edit Mode". Once clicked, the query edit mode changes to give you a full text area in which to write log analytics queries. (This is identical to how the InfluxDB data source lets you write raw queries.)
-
-Once a query is written, the column names are automatically parsed out of the response data. You can then select them in the "X-axis", "Y-axis", and "Split On" dropdown menus, or just type them out.
-
-There are some important caveats to remember:
-
-- You'll want to order your y-axis in the query, eg. `order by timestamp asc`. The graph may come out looking bizarre otherwise. It's better to have Microsoft sort it on their side where it's faster, than to implement this in the plugin.
-
-- If you copy a log analytics query, typically they'll end with a render instruction, like `render barchart`. This is unnecessary, but harmless.
+## Querying the Application Insights Analytics service
```

Comment:
```suggestion
## Query the Application Insights Analytics service
```

Reply:
Here, service is lowercase. In previous headings, "Service" is uppercase. Please make consistent. If "service" is part of the proper name, then uppercase is appropriate. If "service" is generic, not part of the official name, then lowercase is appropriate.